images/singleuser-sample/Dockerfile

```diff
@@ -53,10 +48,16 @@ RUN apt-get update \
         git \
  && rm -rf /var/lib/apt/lists/*
 
-COPY --from=build-stage /build-stage/*.whl /tmp/pre-built-wheels/
+# set env for pip cache,
+# but use ARG to avoid persisting in image
```

Comment:
```suggestion
# set pip's cache directory using this environment variable, and use
# ARG instead of ENV to ensure its only set when the image is built
```